flow/workflows/cdc_flow.go

```diff
@@ -221,11 +220,17 @@ func CDCFlowWorkflow(
 		shared.MirrorNameSearchAttribute: cfg.FlowJobName,
 	}
 
+	var syncCountLimit int
```

Comment:
why temporal state instead of property on config where when 0 it doesn't pause after N batches?

Then user only needs to keep invoking `RESUME MIRROR` after setting value

Reply:
it'd be in the edit mirror dialog